templates/compare/index.html

```diff
@@ -10,204 +10,223 @@
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Why MicroK8s?</h2>
+    </div>
+    <div class="col">
+      <h2 class="p-heading--5">Selecting your lightweight Kubernetes distribution depends on the use case, the resource constraints and your familiarity with the Kubernetes internals.</h2>
+    </div>
+  </div>
+    <div class="row--50-50">
+      <div class="col u-vertically-center u-align--center u-hide--medium u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/deac34ec-cloud.svg" alt="" loading="lazy" class="" />
+      </div>
+      <div class="col">
         <p>MicroK8s is the easiest way to consume Kubernetes as it abstracts away much of the complexity of managing the lifecycle of clusters. Its low-touch UX automates or simplifies operations such as deployment, clustering, and enabling of auxiliary services required for a production-grade K8s environment. Compared to other lightweight distributions MicroK8s is the only option that provides a single-command installation, automatic high availability clustering and automatic updates.</p>
         <p>Robustness is a key primitive, particularly for production environments. Transactional over-the-air patching and security fixes, provided through the snap package as well as self-healing high availability of the Kubernetes control plane services, give MicroK8s the edge for mission-critical workload support.</p>
         <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking &amp; load balancing, storage, service mesh, observability, GPU &amp; FPGA acceleration, multi-cluster management and more - something unique among lightweight Kubernetes distributions.</p>
       </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
-        <img src="https://assets.ubuntu.com/v1/990738e2-kubernetes-cloud.svg" alt="" width="263" height="150" loading="lazy" />
-      </div>
     </div>
   </section>
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <h2>MicroK8s and Kubernetes resources</h2>
-  </div>
-  <div class="row p-divider">
-    <div class="col-8 p-divider__block">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6e184942-Webinar.svg" alt="">
-          <h3 class="p-heading--4 p-heading-icon__title">Webinars on demand</h3>
-        </div>
-        <ul class="p-list is-split">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>MicroK8s and Kubernetes resources</h2>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <p class="p-text--small-caps">Webinars</p>
```

Comment:
```suggestion
        <h3 class="p-text--small-caps">Webinars</h3>
```
I think using Headings here works better from an accessibility standpoint